activation/nipost.go

```diff
@@ -349,7 +378,57 @@ func membersContainChallenge(members []types.Member, challenge types.Hash32) (ui
 	return 0, fmt.Errorf("challenge is not a member of the proof")
 }
 
-func (nb *NIPostBuilder) getBestProof(ctx context.Context, challenge types.Hash32) (types.PoetProofRef, *types.MerkleProof, error) {
+// TODO(mafa): remove after next poet round; https://github.com/spacemeshos/go-spacemesh/issues/4753
```

Comment:
i thought the whole point is simply querying every poet in the config? 
that way there is no need to hard-code anything as long as 111 is in the config?

Reply:
Again as I explained in https://github.com/spacemeshos/go-spacemesh/pull/4752/files/466e168e9486621f54a7675258ca21676f828885#r1278978871

The change to just query every poet in the config is more code change than the current solution and 2nd would cause more unnecessary load on the system.

Only clients that submitted to 110 for round 0 should fetch from 111 when the time comes, since only those will even be able to receive a response.